protocols/request-response/src/lib.rs

```diff
@@ -430,8 +430,9 @@ where
     /// > the `RequestResponse` protocol must either be embedded
     /// > in another `NetworkBehaviour` that provides peer and
     /// > address discovery, or known addresses of peers must be
-    /// > managed via [`Behaviour::add_address`] and
-    /// > [`Behaviour::remove_address`].
+    /// > managed via [`Swarm::add_peer_address`].
```

Comment:
```suggestion
    /// > managed via [`libp2p_swarm::Swarm::add_peer_address`].
```
I think you have to provide the full path for rustdoc.

Reply:
Thanks!
Sorry, sometimes I only have a bit of time per day for OSS and may forget to check the CI
@elenaf9